options/glibc/include/net/ethernet.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _NET_ETHERNET_H
+#define _NET_ETHERNET_H
+
+#include <stdint.h>
+
+// Hmm, linux option?
+#include <linux/if_ether.h>
```

Comment:
Probably a good idea to have this guarded by some linux option defs.

Reply:
But what if, we build mlibc for linux, without the linux-header option, because we use the already installed kernel headers, then that would break right? Also people that don't use linux-headers but do enable glibc will not be able to use this header otherwise. How do you suggest I proceed, guarding the entire header in a linux-header option define?

Reply:
Hmm, do we really need the header include here? The man page for ``ether_aton`` says that ``eth_addr`` is defined as ``uint8_t ether_addr_octet[6];``. That would solve this issue.